resources/tentative/complex-static-html/IMPACT_ANALYSIS.md

```diff
@@ -0,0 +1,124 @@
+* **React-Big**: The TodoMVC-React benchmark embedded in a big static HTML page, the benchmark introduced in this PR.
+* *React-CSS*: The original TodoMVC-React benchmark with a lot of CSS rules.
+* *React-Big-{depth}(0,10,30)*: The TodoMVC-React benchmark embedded in a node of depth {depth} in a big static HTML page filled with buttons.
+* *React-Big-{depth}-CSS(0,10,30)*: The TodoMVC-React benchmark embedded in a node of depth {depth} in a big static HTML page filled with buttons, with extra CSS rules.
+* *React-Mail*: An earlier iteration of *TodoMVC-React-Big* built with different tooling, and which the group agreed was a good starting point for the workload.
+
+Each benchmark was ran 50 times on each browser. The following boxplots show the distribution of the results. The x-axis is milliseconds, the y-axis is the benchmark ran.
+
+### Total ms analysis
+
+<p align = "center" background-color="white">
+<img src="impact-analysis-images/boxplots.png" alt="boxplots" width="800"/>
+</p>
+
+A quick overview to these boxplots reveals that the different additions affect the different browsers differently. With **React-Big** and *React-Mail* having the biggest impact accross all browsers.
+
+To dive deeper into the specifics of each test lets analyze the boxplots visualizations below
+
+> How to interpret the heatmaps: The heatmap shows the p-value of the mannWhitney test between the two benchmarks. A lighter color represents there is statistically significant evidence that the benchmark in {row} is faster than the benchmark in {column}.
+
+### Chrome
+<p align = "center" background-color="white">
+<img src="impact-analysis-images/total-chrome.png" alt="heatmap-chrome" width="800"/>
+</p>
+
+On Chrome, there is an impact by embedding the TodoMVC in a big DOM plus a styles UI, like **React-Big** and *React-Mail* do. The differences between the variations of *React-Big-{depth}* and *React-Big-{depth}-CSS* suggest that the inclusions of the extra CSS rules also has a significant impact on performance.
+
+### Firefox
+<p align = "center" background-color="white">
+<img src="impact-analysis-images/total-firefox.png" alt="heatmap-chrome" width="800"/>
+</p>
+
+On Firefox, the impact from *React-Mail* is not clear here, whereas the impact of **React-Big* is the clear. There seems to be an impact caused by the CSS and the size of the DOM too.
+
+### Safari
+<p align = "center" background-color="white">
+<img src="impact-analysis-images/total-safari.png" alt="heatmap-chrome" width="800"/>
+</p>
+
+On Safari, the performance of the benchmark is impacted by every variation. The DOM size has an impact that is worsen when adding CSS.
+
+### Per-step analysis
+
+The benchmarks impact the different test steps differently for each browser, as we can see in the per-step heatmaps below.
+
+### Chrome
+<p align = "center" background-color="white">
+<img src="impact-analysis-images/step-chrome.png" alt="heatmap-chrome" width="800"/>
```

Comment:
Please don't include this kind of design doc as a part of the benchmark content.
This kind of stuff should be kept in an external document somewhere instead.